tests/integration/TEC/Tickets/Admin/All_Tickets/__snapshots__/PageTest__test_render_tec_tickets_all_tickets_page__0.snapshot.html

```diff
@@ -2,11 +2,33 @@
 	<h1>
 	All Tickets</h1>
 <form id="tec-tickets-all-tickets-form" method="get">
+	<input type="hidden" name="page" value="tec-tickets-all-tickets" />
 	<input type="hidden" id="_wpnonce" name="_wpnonce" value="WP_NONCE" /><input type="hidden" name="_wp_http_referer" value="" />	<div class="tablenav top">
 
 				<div class="alignleft actions bulkactions">
 					</div>
-			<div class='tablenav-pages one-page'><span class="displaying-num">3 items</span>
+			<div class="alignleft actions">
+	<select name="status-filter" id="tec-tickets-all-tickets-status-filter">
+			<option value="active"  selected='selected'>
+			Active Tickets		</option>
+			<option value="past" >
+			Past Tickets		</option>
+			<option value="upcoming" >
+			Upcoming Tickets		</option>
+			<option value="discounted" >
+			Discounted Tickets		</option>
+			<option value="all" >
+			All Tickets		</option>
```

Comment:
I guess every status option is being printed. Not sure if it would be a better experience to print only what is present in the DB

Reply:
See my comment about the statuses above. They are not in the DB. Most of them are based on the start/end sales dats (past, upcoming, active, etc.).